### ISSUE TITLE
301 redirects shouldn't be followed

### ISSUE THREAD

**Issue by denisvlr** [skoczen/django-seo-js#9]
I ran into an issue while trying to handle 301 redirects:

When the prerender server (either self-hosted or on prerender.io) sends a 301 redirection, django-seo-js responds with a 200, it seems to follow redirections.

IMHO this is not the expected behaviour, Google needs to know that the url has changed. I think the simple fix should be to add `allow_redirects=False` in the call to `request.get` :
```
r = self.requests.get(render_url, headers=headers, allow_redirects=False)
```
Is there any scenario where we want redirects to be followed? If so, we can add a global setting, otherwise this should be disabled by default.

**Comment by mattrobenolt** [comment]
+1

**Comment by skoczen** [comment]
I'm +1 to this, just need free cycles to get it written and relased. If someone has time to put a PR together, that'd help - thanks!

**Comment by mattrobenolt** [comment]
@skoczen This is easy, but will conflict with my #10. I'll add this after you merge that.

**Comment by skoczen** [comment]
I'll just pull it in to the 0.3 release - thanks for all the work on this!

**Comment by skoczen** [comment]
This is included in [0.3](https://github.com/skoczen/django-seo-js#030---feb-5-2015), and out - let me know if you hit any snags!
